ci-operator/step-registry/openshift/e2e/vsphere/upi/zones/openshift-e2e-vsphere-upi-zones-workflow.yaml

```diff
@@ -0,0 +1,11 @@
+workflow:
+  as: openshift-e2e-vsphere-upi-zones
+  steps:
+    pre:
+    - chain: upi-vsphere-pre-zones
+    test:
+    - ref: openshift-e2e-test
```

Comment:
Is there a test which ensures machines can use all clusters in `failureDomains`?

Reply:
Not explicitly, but the terraform distributes the 4 compute nodes among the 4 failure domains.  I'll take a note to investigate this and see if we need to add a more complete test.  Thanks for reviewing the PR!